### PR TITLE
Addind Jonathan at Credits

### PR DESCRIPTION
**Pull request by jsantos93**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
 - Insert
 - Remove
 - Print
 - SimpleRotate
 - DoubleRotate (Needs SimpleRotate)
 - Search
+
+## Credits
+
+- Jonathan
```
Comment (GuilhermeAO) [inline README.md]: Welcome

**Review by GuilhermeAO — APPROVED** [review]

**Review by OtacilioN — CHANGES_REQUESTED** [review]
Hey Jonathan, awesome to see your name in the credits, but you should add your surname too.

File: README.md [hunk 2]
```diff
@@ -13,4 +13,4 @@ A simple implementation of an AVL Tree
 
 ## Credits
 
-- Jonathan
+- Jonathan Santos
```
Comment (OtacilioN) [inline README.md]: Now is great!

**Review by OtacilioN — APPROVED** [review]
Awesome see you here!

**Pull request merged**